### PR TITLE
Let admins configure an "announcement" via SiteMode object

### PR DESCRIPTION
**Pull request by ejucovy**
* Add announcement-related fields on `SiteMode`: headline, body, link url
* Announcement will only appear (in red with megaphones) if headline is filled out
* An `announcement_page_regex` field controls which pages the announcement should appear on, so e.g. announcement can be enabled site-wide with link to a `/watch/` flatpage, but without the announcement showing up on the `/watch/` page itself

### CODE REVIEW

File: opendebates/static/less/base.less [hunk 1]
```diff
@@ -168,12 +168,54 @@ body > .container {
...
+      .full({
+        content: "";
+        background-image: url(/static/images/megaphone-right.png);
+        right: 15px;
+        bottom: 0;
+        height: 130px;
+        width: 176px;
+        position: absolute;
+      });
+    }
```
Comment (ironfroggy) [inline opendebates/static/less/base.less]: Looks like these get applied now to the Old Browser Upgrade banner that was recently added. Did you want to do that, or would you like to restrict these only to the announcement banner?

Comment (ironfroggy) [inline opendebates/static/less/base.less]: Everything else looks fine, I just want to be sure about the impact on existing banners before merging.

Comment (ejucovy) [inline opendebates/static/less/base.less]: Yeah, I figured I might as well apply these to the upgrade banner too -- though I didn't actually think to check how it looks on old browsers.  Do you think that's risky?  Would these styles be likely to mangle the upgrade banner on the browsers that need to see it?

Comment (ironfroggy) [inline opendebates/static/less/base.less]: looks fine. just wanted to be sure it was intended

**Pull request merged**